src/main/java/org/generationcp/ibpworkbench/germplasm/SaveGermplasmListDialog.java

```diff
@@ -1,262 +0,0 @@
-/*******************************************************************************
```

Comment:
same here @gelli-leafnode 
Not sure how they are related to Germplasm Details?

Reply:
Hi @clarysabel , 

The SaveGermplasmListDialog doesn't have declaration to any classes, it was only used in GermplasmButtonClickListener w/c was deleted as well because was only declared in deleted classes: 
  GermplasmDerivativeNeighborhoodComponent
  GermplasmDetails 
  GermplasmMaintenanceNeighborhoodComponent 
  SaveGermplasmListDialog